conftest.py

```diff
@@ -7,8 +7,8 @@
 settings.register_profile("travis-ci" , settings(max_examples = 1000, deadline=300))
 settings.register_profile("hard"      , settings(max_examples = 1000))
 settings.register_profile("dev"       , settings(max_examples =   10))
-settings.register_profile("hard_nocov", settings(max_examples = 1000, use_coverage=False))
-settings.register_profile("dev_nocov" , settings(max_examples =   10, use_coverage=False))
+settings.register_profile("hard_nocov", settings(max_examples = 1000))
+settings.register_profile("dev_nocov" , settings(max_examples =   10))
```

Comment:
How can we deactivate coverage now? Or we don't care anymore? In any case, we have duplicated profiles now.

Reply:
```
settings._define_setting(
    "use_coverage",
    default=not_set,
    deprecation_message="""
use_coverage no longer does anything and can be removed from your settings.
""",
    deprecated_since="2017-09-14",
    description="""
A flag to enable a feature that no longer exists. This setting is present
only for backwards compatibility purposes.
""",
)
```
Seems that we couldn't deactivate for some time already!